Modules/AirCastingStyling/Sources/AirCastingStyling/FollowButtonStyle.swift

```diff
@@ -14,7 +14,7 @@ public struct FollowButtonStyle: ButtonStyle {
         configuration
             .label
             .foregroundColor(.white)
-            .frame(width: 82, height: 23, alignment: .center)
+            .frame(width: 82, height: 29, alignment: .center)
             .background(Color.accentColor)
             .font(.muli(size: 13, weight: .semibold))
             .cornerRadius(11)
```

Comment:
Please fix here the cornerRadius as it changed and does not match map and graph button anymore